.github/workflows/test_lottie.yaml

```diff
@@ -62,6 +62,8 @@ jobs:
 
           grcov $(pwd)/tests/unit/CMakeFiles/lottie_tests.dir -s ${{github.workspace}} \
               -t lcov --branch --ignore-not-existing > ${{github.workspace}}/lottie_lcov.info
+        ignore:
+          - "tests/*"
```

Comment:
I'm thinking let's also not do codecov for the lottie tests and just have it for the unit tests so that we only have one workflow uploading to codecov. Once we understand that, we can look at what happens when two different workflows are uploading to codecov. What do you think?